avocado/core/varianter.py

```diff
@@ -45,9 +45,31 @@ def generate_variant_id(variant):
     :return: String compounded of ordered node names and a hash of all
              values.
     """
+    def variantname(variant):
```

Comment:
In that case, I suggest changing this method name: `get_variant_name()` or something like this.

Reply:
tnx!, fixed